CONTRIBUTING.md

```diff
@@ -59,3 +59,23 @@ If you'd like to contribute to the design or front-end, feel free to contribute
 Documentation is automatically synced from master to the [Gutenberg Documentation Website](https://wordpress.org/gutenberg/handbook/) every 15 minutes.
 
 To add a new documentation page, you'll have to create a Markdown file in the [docs](https://github.com/WordPress/gutenberg/tree/master/docs) folder and add an item to the [manifest file](https://github.com/WordPress/gutenberg/blob/master/docs/manifest.json).
+
+### In Code Documentation
+
+This project uses [JSDoc](http://usejsdoc.org/) for doc blocks. Inline code should be used for describing individual pieces in detail. The in code documentation will hopefully in the future be repurposed for [the WordPress code reference](https://developer.wordpress.org/reference/). If you would like to help contribute to the in code documentation, try adding a JSDoc header to something that needs one! It would be greatly appreciated!
+
+In code documentation priorities:
```

Comment:
How did you come to this prioritization? Should we set prioritization expectations, or merely that all things are to be documented (prioritization implies that you don't need to document everything)?

Reply:
> How did you come to this prioritization?

I believe I went off of what was already documented, and what seemed undocumented (higher undocumented rate lower on prioritization). We can take that part out.

I wrote this for feedback, I have no idea if there should be prioritization. I would probably be most in favor of documenting everything so we can potentially extract the docs out programmatically, but we did not come to a consensus in the JS meeting. Without your and Riad's buy in, since ya'll are doing the vast bulk of the work, it would be unfair to try to impose anything on you two.